discovery.md

```diff
@@ -224,11 +231,85 @@ The following sections define the attributes that appear in a Service entity.
   See the Primer for more information.
 
 - Constraints:
-  - REQUIRED
+  - conditionally REQUIRED. See above.
   - MUST be a valid UUID per RFC4122.
 - Examples:
   - `bf5ff5cc-d059-4c79-a89a-2513e45a1340`
 
+##### discversion
+
+- Type: `Integer`
+- Description: The Discovery Endpoint's version number of this Service Entry.
+  A monotonically increasing number, starting with `1` that is
+  increased by `1` each time an update is performed on this Discovery
+  Endpoint's entry. Note: this version value is not related to the version
+  of the Service itself representated by this entry. This value is meant to
+  be used to know when the Discovery Enpoint's metadata about the Service
+  has been updated.
+
+  While this attibute is REQUIRED to be part of the the Service's description
+  when returned from a Discovery Endpoint, depending on the action from the
+  client it might not be mandatory to appear in a client's request. For a
+  "create" operation, this attribute MUST NOT appear. For an "import" type
+  of operation, where it is important for the entry to retain a previously
+  defined value, this attribute MUST appear
+
+  A client MAY choose to include this attribute on an "update" operation.
+  If present then the Discovery Endpoint MUST ensure that the value
+  on the incoming request matches the current value within the Discovery
+  Endpoint, and if it does not then an error MUST be generated and the Service
+  MUST NOT be updated. However, if the incoming request does not include this
```

Comment:
Doesn't an "update" operation alter the metadata for the Service? If so, is the expectation that a subsequent `GET /services/{service}` would have a discversion that has increased from the provided value after this update?

Reply:
yes, a subsequent GET would have a new discversion value